packages/lib-content/src/screens/Resources/Resources.js

```diff
@@ -0,0 +1,269 @@
+                      key='zooniverse-home-page'
+                      href='https://www.zooniverse.org'
+                    />
+                  ]}
+                />
+              </Text>
+            </Box>
+            <Paragraph margin='0'>
+              <Trans
+                i18nKey='Resources.cite.paragraphs.two'
+                t={t}
+                components={[
+                  <Anchor
+                    key='publications-form'
+                    href='https://docs.google.com/forms/d/e/1FAIpQLSdbAKVT2tGs1WfBqWNrMekFE5lL4ZuMnWlwJuCuNM33QO2ZYg/viewform'
+                  />,
+                  <Anchor
+                    key='publications-page'
+                    href='https://www.zooniverse.org/about/publications'
+                  />
+                ]}
+              />
+            </Paragraph>
+            <Paragraph margin={{ top: 'small', bottom: 'medium' }}>
+              <Trans
+                i18nKey='Resources.cite.paragraphs.three'
+                t={t}
+                components={[
+                  <Anchor
+                    key='contact-us'
+                    href='https://www.zooniverse.org/about#contact'
+                  />
+                ]}
+              />
+            </Paragraph>
+
+            {/** FOR MEDIA */}
+            <Text
+              size='1.13rem'
+              weight='bold'
+              color={{ light: 'black', dark: 'white' }}
+            >
+              {t('Resources.cite.subheadings.two')}
+            </Text>
+            <Paragraph margin={{ top: 'small', bottom: '0' }}>
+              {t('Resources.cite.paragraphs.four')}
+            </Paragraph>
+            <Paragraph margin={{ vertical: 'small' }}>
+              {t('Resources.cite.paragraphs.five')}
+            </Paragraph>
+            <Paragraph margin={{ top: '0', bottom: 'medium' }}>
+              <Trans
+                i18nKey='Resources.cite.paragraphs.six'
+                t={t}
+                components={[
+                  <Anchor
+                    key='contact-us'
+                    href='https://www.zooniverse.org/about#contact'
+                  />
+                ]}
+              />
+            </Paragraph>
+
+            {/** BRAND RESOURCES */}
+            <SpacedHeading
+              align='center'
+              margin={{ top: '0', bottom: 'small' }}
+              size='1.5rem'
+              textAlign='center'
+            >
+              {t('Resources.brand.heading')}
+            </SpacedHeading>
+            <StyledList>
+              <Box
+                as='li'
+                direction='row'
+                align='center'
+                gap='small'
+                margin={{ bottom: 'small' }}
+              >
+                <ResourceLogo>
+                  <ZooniverseLogo
+                    size='45px'
+                    color='#005D69' // 'neutral-1' won't work here for some reason
+                    id='link to logo brand resources'
```

Comment:
Same as above comment on `id` attribute. The text for this one doesn't fit either as it's the Zoonivese logo.